PIC_IoT_Central.md

```diff
@@ -89,7 +89,7 @@ IoT Central allows you to create an application dashboard to monitor the telemet
 
     - Refer to the [Dev Tools Installation](./Dev_Tools_Install.md) procedure and confirm that `Azure CLI`, `Python`, and `pyazureutils` have all been installed. The device certificate should reside in the `.microchip_iot` folder which was generated by the IoT Provisioning Tool, so `pyazureutils` will look in that folder to grab the correct certificate used during the attestation process required for connecting a device to your IoT Central application
     
-    - Launch a `Command Prompt` or `PowerShell` window and then execute the following command (where <YOUR_CUSTOM_APP_NAME> is the unique `Application name` which was set in the previous step).  You will be prompted to log into your Microsoft Azure account if you are not currently signed in
+    - Launch a `Command Prompt` or `PowerShell` window and then execute the following command line (where <YOUR_CUSTOM_APP_NAME> is the unique `Application name` which was set in the previous step). If this command fails to execute, try specifying the `Application URL` for <YOUR_CUSTOM_APP_NAME>. You may be prompted to log into your Microsoft Azure account if you are not currently signed in
 
         ```bash
         pyazureutils iotcentral register-device --application-name <YOUR_CUSTOM_APP_NAME>
```

Comment:
should this be changed to
```suggestion
        pyazureutils iotcentral register-device --application-name <YOUR_CUSTOM_APP_URL>
```

Reply:
No, cuz the URL has to go in place of <YOUR_CUSTOM_APP_NAME> in the command line that's shown